src/quacc/calculators/espresso/espresso.py

```diff
@@ -298,7 +298,10 @@ def __init__(
         self._user_calc_params = {}
 
         template = template or EspressoTemplate("pw")
-        self._bin_path = str(SETTINGS.ESPRESSO_BIN_PATHS[template.binary])
+        full_path = Path(
+            SETTINGS.ESPRESSO_BIN_DIR, SETTINGS.ESPRESSO_BIN_PATHS[template.binary]
+        )
```

Comment:
This seems to conflict with the `ESPRESSO_BIN_PATHS` description, which implies that it's the full path to each executable. Perhaps some further clarification is in order.

Reply:
ESPRESSO_BINS or ESPRESSO_BIN_NAMES ? I am open to other suggestions

Reply:
Depends on your intention. I think originally you wanted to have it be possible for every executable to have a custom path. That was the motivation for having a single `ESPRESSO_BIN_PATHS`. But if you think that use case is highly unlikely, then a `ESPRESSO_BINARIES` would probably be fine.

Reply:
I would still like to be able to have a different path for each binary although I would like it to be compatible with the config file and CLI: quacc set ESPRESSO_BIN_PATHS['pw'] = ... but I guess that's out of reach

Reply:
There's definitely a way to do that kind of thing with `typer` (albeit likely with different syntax) but yeah, not implemented currently.